components/molecule/photoUploader/src/index.js

```diff
@@ -434,6 +436,9 @@ MoleculePhotoUploader.propTypes = {
   /** Text showed at error notification when some file of the initialPhotos fails */
   errorInitialPhotoDownloadErrorText: PropTypes.string.isRequired,
 
+  /** Text showed at error notification when the saveImages endpoint returns an error  */
+  errorSaveImageEndpoint: PropTypes.string.isRequired,
```

Comment:
Are you sure you want to do this prop required? It breaks retro compatibility. I think it is not needed.

Reply:
Good observation Aitor! I took out the required one and added a default message to it.